src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -13,5 +13,6 @@ export interface ResourceIF {
     filingTypeCode: FilingCodes,
     priority?: boolean
   },
+  typeChangeInfo?: string
```

Comment:
Would it make sense to create a "change" object to nest this under? Maybe in the future when there is more than 1 unique change property?

Reply:
Updated!